packages/fast-components-react-msft/src/toggle/index.ts

```diff
@@ -0,0 +1,6 @@
+import * as React from "react";
+import {  IFoundationProps, IToggleClassNameContract, IToggleProps, Toggle } from "@microsoft/fast-components-react-base";
```

Comment:
can you remove the spacing between the brackets and the imports?

Reply:
UPDATE: Leave spacing here due to the linter not being able to lint this. It should now lint import and exports to include space. Within the ts files themselves, there should be no spaces.